[P2] Two pre-existing test failures on main: test_min_volume_default and polymarket/maker-rebate-bot unbuffered-output

## Summary

Two unrelated tests on `main` are failing. Neither is a production regression — both are drift between tests and code/convention. They should be fixed so `pytest tests/` is green on `main`.

## Failure 1: `tests/test_market_selection_gates.py::test_min_volume_default_is_1000`

```
assert 5000.0 == 1000.0
 +  where 5000.0 = float('5000.0')
```

**Root cause**: PR #295 raised `min_volume` default from `1000.0` -> `5000.0` in both `polymarket/bot/scripts/agent.py` and `config.example.json` (intentional — filter dead-book markets). The test at `tests/test_market_selection_gates.py:110-114` was not updated alongside it.

**Fix**: Update the test assertion to match the current $5,000 floor and rename to `test_min_volume_default_is_5000` to reflect reality.

## Failure 2: `tests/test_unbuffered_output.py::test_agent_has_unbuffered_output[polymarket/maker-rebate-bot]`

```
AssertionError: polymarket/maker-rebate-bot/scripts/agent.py is missing the
unbuffered output fix. Add the PYTHONUNBUFFERED / reconfigure(line_buffering=True)
block after 'import sys' so piped output is visible immediately.
```

**Root cause**: `polymarket/maker-rebate-bot/scripts/agent.py` is missing the standard unbuffered-stdout block that the convention (enforced by `tests/test_unbuffered_output.py`) requires on every trading skill's `agent.py`.

**Fix**: Add the canonical block after `import sys`:

```python
# --- Force unbuffered stdout so piped/background output is visible immediately ---
if not sys.stdout.isatty():
    os.environ.setdefault("PYTHONUNBUFFERED", "1")
    sys.stdout.reconfigure(line_buffering=True)
    sys.stderr.reconfigure(line_buffering=True)
# --- End unbuffered stdout fix ---
```

## Acceptance

- [ ] `pytest tests/test_market_selection_gates.py tests/test_unbuffered_output.py` is green on `main`.
- [ ] No other behavior changes.

## Severity

P2 — no production impact, but makes CI noisy and hides real regressions.

Taariq Lewis, SerenAI, Paloma, and Volume at https://serendb.com
Email: hello@serendb.com

--- a/polymarket/maker-rebate-bot/scripts/agent.py
+++ b/polymarket/maker-rebate-bot/scripts/agent.py
@@ -9,6 +9,14 @@ import math
 import os
 import sys
 import time
+
+# --- Force unbuffered stdout so piped/background output is visible immediately ---
+if not sys.stdout.isatty():
+    os.environ.setdefault("PYTHONUNBUFFERED", "1")
+    sys.stdout.reconfigure(line_buffering=True)
+    sys.stderr.reconfigure(line_buffering=True)
+# --- End unbuffered stdout fix ---
+
 from concurrent.futures import ThreadPoolExecutor, as_completed
 from dataclasses import dataclass, replace
 from datetime import datetime, timezone

--- a/tests/test_market_selection_gates.py
+++ b/tests/test_market_selection_gates.py
@@ -107,11 +107,11 @@ def test_min_volume_filter_exists() -> None:
     assert "min_volume" in source, "agent.py missing min_volume config"
 
 
-def test_min_volume_default_is_1000() -> None:
+def test_min_volume_default_is_5000() -> None:
     source = _read(BOT_AGENT)
     match = re.search(r"['\"]min_volume['\"],\s*([\d.]+)", source)
     assert match, "Cannot find min_volume default in agent.py"
-    assert float(match.group(1)) == 1000.0
+    assert float(match.group(1)) == 5000.0
 
 
 def test_min_volume_in_config_example() -> None:

